config/test.exs

```diff
@@ -5,7 +5,12 @@ config :fishjam,
   port: 4002,
   server_api_token: "development",
   webrtc_metrics_scrape_interval: 50,
-  room_metrics_scrape_interval: 1
+  room_metrics_scrape_interval: 1,
+  feature_flags: [
+    # TODO: Enable this flag here once we start using it in production
+    request_routing_enabled?: false
```

Comment:
I feel like this is unecessary since we have this in `runtime.exs` and also default value is set there.